resources/integration_test/tests/columns_all_filter.yml

```diff
@@ -3,15 +3,15 @@ tests:
     test_definition: [ "sqlcolumnsw", "integration_test", 0, "", 0, "%o%", 3, "%i%", 3 ]
     db: integration_test
     expected_result:
-      - ["integration_test", null, "a_non_lexicographic_field_order", "_id", "4", "int", 10, "4", 0, 10, 1, "", null, "4" , null , 0, 4, "YES"]
```

Comment:
Which change in ADF caused these values to get updated?

Reply:
https://github.com/10gen/mongohouse/commit/d768d7aef9772a7926dd01cd61508e633dcd79af